x-pack/plugins/ml/server/lib/register_sameple_data_set_links.ts

```diff
@@ -10,8 +10,8 @@ import type { HomeServerPluginSetup } from '@kbn/home-plugin/server';
 import type { MlFeatures } from '../types';
```

Comment:
Would it be possible to correct the typo on the name of this file `sameple` in this PR?

Reply:
Fixed in https://github.com/elastic/kibana/pull/166477/commits/2ebb3a1d21e71cd6454072a6c29e3892010a7b84